src/SelfService/Infrastructure/Api/Capabilities/CapabilityController.cs

```diff
@@ -910,4 +924,52 @@ public async Task<IActionResult> Join(string id)
         }
         return Ok();
     }
+
+    [HttpPost("{id}/invitations")]
+    [ProducesResponseType(typeof(CapabilityDetailsApiResource), StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest, "application/problem+json")]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status401Unauthorized, "application/problem+json")]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound, "application/problem+json")]
+    public async Task<IActionResult> CreateInvitations([FromRoute] string id, [FromBody] InvitationsRequest request)
+    {
+        if (!User.TryGetUserId(out var userId))
+        {
+            return Unauthorized();
+        }
+
+        if (!CapabilityId.TryParse(id, out var capabilityId))
+        {
+            return BadRequest(
+                new ProblemDetails { Title = "Invalid CapabilityId provided", Status = StatusCodes.Status400BadRequest }
+            );
+        }
+
+        if (!await _authorizationService.CanInviteToCapability(userId, capabilityId))
+        {
+            return Unauthorized();
+        }
+
+        var capability = await _capabilityRepository.Get(capabilityId);
+        if (capability is null)
+        {
+            return NotFound(
+                new ProblemDetails
+                {
+                    Title = "Capability not found.",
+                    Detail = $"A capability with id \"{id}\" could not be found."
+                }
+            );
+        }
+        if (request.Invitees != null)
+        {
+            return Ok(
+                await _invitationApplicationService.CreateCapabilityInvitations(
+                    invitees: request.Invitees,
+                    inviter: userId,
+                    capability: capability
+                )
```

Comment:
Where do you see us throwing an error?
I am not sure I see it at least

Reply:
My bad, no exceptions are thrown indeed. Would be cool with a plugin that could show if a statement can throw exceptions otherwise i am always worried in these controllers